qtism/data/rules/SetOutcomeValue.php

```diff
@@ -43,6 +43,13 @@
  */
 class SetOutcomeValue extends QtiComponent implements OutcomeRule, ResponseRule
 {
+    /**
+     * FIXME: This constant is accessed directly in Tao ImportService.
+     * Every other places are using hard-coded string, which is even worse.
+     * It shouldn't be disclosed and another way of selecting elements must be found.
+     */
+    const CLASS_NAME = 'setOutcomeValue';
```

Comment:
```suggestion
    public const CLASS_NAME = 'setOutcomeValue';
```

Reply:
Sorry but I won't add the access modifier here, since it wouldn't parse in PHP 7.0.